data/schema.js

```diff
@@ -1,4 +1,4 @@
-var getbabelRelayPlugin = require('babel-relay-plugin');
-var schema = require('./schema.json');
+var getbabelRelayPlugin = require('babel-relay-plugin')
+var schema = require('./schema.json')
 
-module.exports = getbabelRelayPlugin(schema.data);
\ No newline at end of file
+module.exports = getbabelRelayPlugin(schema.data, { substituteVariables: true })
```

Comment:
Ooohhhh, I see. Enabling this just fakes the possibility of being able to use `${userID}` in fragments and translates it to just `$userID`. Hehehe, fun, but we don’t need that indeed.